src/mapValues.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Maps values of `object` and keeps the same keys.
+ * @param object the object to map
+ * @param fn the mapping function
+ * @signature
+ *    R.mapValues(object, fn)
+ * @example
+ *    R.mapValues({a: 1, b: 2}, (value, key) => value + key) // => { a: 1a, b: 2b }
+ * @data_first
+ * @category Object
+ */
+export function mapValues<T extends object, S>(
+  object: T,
+  fn: (value: T[keyof T], key: keyof T) => S
+): Record<keyof T, S>;
+
+/**
+ * Maps values of `object` and keeps the same keys.
+ * @param fn the mapping function
+ * @signature
+ *    R.mapKeys(fn)(object)
+ * @example
+ *    R.pipe({a: 1, b: 2}, R.mapValues((value, key) => value + key)) // => { a: 1a, b: 2b }
```

Comment:
quotes are missing in `{ a: 1a, b: 2b }`

Reply:
also here: 7e309318